crates/db_schema/src/impls/instance.rs

```diff
@@ -201,6 +201,8 @@ impl Instance {
       .left_join(federation_blocklist::table)
       .left_join(federation_allowlist::table)
       .left_join(federation_queue_state::table)
+      // Show recently updated instances and those with valid metadata first
+      .order(instance::updated_at.desc().nulls_last())
```

Comment:
If by valid metadata, don't you still want the secondary sort to be `software.nulls_last()` ? 

If not, then feel free to merge.

Reply:
Ah misunderstood how it works, fixed now.